CONTRIBUTING.md

```diff
@@ -1,22 +1,79 @@
-This is a [Next.js](https://nextjs.org/) project bootstrapped with [`create-next-app`](https://github.com/vercel/next.js/tree/canary/packages/create-next-app).
+# Contributing to the Ferdium Website
 
-## Getting Started
+:tada: First off, thanks for taking the time and your effort to make Ferdium better! :tada:
+
+## Table of contents
+
+<!-- TOC depthFrom:2 depthTo:2 withLinks:1 updateOnSave:1 orderedList:0 -->
+
+- [Table of contents](#table-of-contents)
+- [Code of Conduct](#code-of-conduct)
+- [How do I contribute?](#how-do-i-contribute)
+	- [Setup your development environment](#setup-your-development-environment)
+		- [Node.js, pnpm](#nodejs-pnpm)
+		- [Git](#git)
+	- [Clone the repository](#clone-the-repository)
+	- [Install dependencies](#install-dependencies)
+	- [Add your modifications](#add-your-modifications)
+	- [Build the website locally](#build-the-website-locally)
+- [Learn More](#learn-more)
+
+<!-- /TOC -->
+
+## Code of Conduct
+
+This project and everyone participating in it is governed by the [Ferdium Code of Conduct](https://raw.githubusercontent.com/ferdium/ferdium-app/develop/CODE_OF_CONDUCT.md). By participating, you are expected to uphold this code.
+Please report unacceptable behavior to [hello@ferdium.org](mailto:hello@ferdium.org).
+
+## How do I contribute?
+
+### Setup your development environment
+
+#### Node.js, pnpm
+
+You will need a working installation of Node.js, with the `pnpm` package manager.
+It is advised to use the version recommended in the [`ferdium-app` repository](https://github.com/ferdium/ferdium-app/blob/develop/CONTRIBUTING.md#nodejs-npm-pnpm).
+
+_Note:_ You can choose any version manager to manage multiple versions of `node` and `npm`. For eg, [nvm](https://github.com/nvm-sh/nvm) or [asdf](https://github.com/asdf-vm/asdf).
+
+#### Git
+
+The version [2.23.0](https://github.com/git-for-windows/git/releases/tag/v2.23.0.windows.1) for Git is working fine for development. You can then use the console from Git to do the development procedure.
```

Comment:
I would much rather point to the main git repo/url, and not to the clone for Windows.
https://git-scm.com/

Reply:
That's right. In fact, I copy-pasted it [from the main repo](https://github.com/ferdium/ferdium-app/blob/16e66963f7907bacc42c189d41f3db0723864068/CONTRIBUTING.md#git), which as the same problem.

I'm wondering: is the `git` information really needed? If a developer is on GitHub and reads this file, doesn't he already know about git?

```suggestion
The version [2.23.0](https://git-scm.com/downloads) for Git is working fine for development. You can then use the console from Git to do the development procedure.
```

Reply:
ok - we should change this in the main repo as well